dropwizard/README.md

```diff
@@ -0,0 +1,108 @@
+Dropwizard Armeria Bundle
+========================
+
+A bundle for delegating application traffic through [Armeria](https://line.github.io/armeria/index.html) endpoints in Dropwizard applications.
+
+Dependency Info
+---------------
+```xml
+<dependency>
+    <groupId>com.linecorp.armeria</groupId>
+    <artifactId>armeria-dropwizard</artifactId>
+    <version>0.95.0</version>
+</dependency>
+```
+
+Usage
+-----
+Add an `ArmeriaBundle` to your [Application](https://javadoc.io/static/io.dropwizard/dropwizard-core/1.3.16/io/dropwizard/Application.html) class.
+
+```java
+@Override
+public void initialize(Bootstrap<MyConfiguration> bootstrap) {
+    // ...
+    final ArmeriaBundle bundle = new AbstractArmeriaBundle<DropwizardArmeriaConfiguration>() {
+        @Override
+        public void onServerBuilderReady(final ServerBuilder builder) {
+            builder.service("/", (ctx, res) -> HttpResponse.of(MediaType.HTML_UTF_8, "<h2>It works!</h2>"));
+            builder.service("/armeria", (ctx, res) -> HttpResponse.of("Hello, Armeria!"));
+
+            builder.annotatedService(new HelloService());
+
+            // You can also bind asynchronous RPC services such as Thrift and gRPC:
+            // builder.service(THttpService.of(...));
+            // builder.service(GrpcService.builder()...build());
+        }
+    };
+    bootstrap.addBundle(bundle);
+}
+```
+
+Configuration
+-------------
+This bundle introduces a new Dropwizard `ServerFactory` of type `armeria` that implements the same contract of the [`simple` ServerFactory](https://www.dropwizard.io/en/stable/manual/configuration.html#simple) as well as [HTTP(S) `ConnectorFactory`](https://www.dropwizard.io/en/stable/manual/configuration.html#http) implementations of `armeria-http` or `armeria-https`. 
+
+A minimal configuration file looks like so. 
+
+```yaml
+server:
+  type: armeria
+  connector:
+    type: armeria-http
+    port: 8080
```

Comment:
Currently, the `connector` property has a default, so this isn't the most minimal config

Reply:
Question: Do Dropwizard users expect a Dropwizard server to have a default port of 8082? What is the most seamless behavior?

Reply:
8080 is the default. I don't have a compelling reason to keep it at 8082